src/src.pro

```diff
@@ -246,6 +246,7 @@ HEADERS += \
         features/featureinappaccountcreate.h \
         features/featureinappauth.h \
         features/featureinapppurchase.h \
+        features/featureincludeallnetworks.h \
```

Comment:
Add it only in the `ios` block.

Reply:
Plus you need to create the feature in `featurelist.cpp` but only `#ifdef MVPN_IOS`